gatekeeper/bootstrap/aws/aws.go

```diff
@@ -0,0 +1,204 @@
+// Package aws provides Gatekeeper authentication for AWS
+package aws
```

Comment:
I think it's probably safe to remove the logic on the client side. It does a pre-flight validation on the instance signature before it sends it to the Gatekeeper, but that is probably unneeded since the Gatekeeper does this check anyway.